libs/utils/utils.js

```diff
@@ -517,6 +517,11 @@ export async function loadArea(area = document) {
 
   // Post section loading on document
   if (isDoc) {
+    const georouting = getMetadata('georouting') || config.geoRouting;
+    if (georouting !== 'off') {
```

Comment:
Seeing how this is working in the real world, I think we need to change this to be opt-in. We have live sites this will immediately impact, and ideally it should not.

Reply:
alright, I'll change that.